nestkernel/conn_builder.cpp

```diff
@@ -74,75 +71,145 @@ nest::ConnBuilder::ConnBuilder( NodeCollectionPTR sources,
   updateValue< bool >( conn_spec, names::make_symmetric, make_symmetric_ );
```

Comment:
Does `make_symmetric` work with colocated synapses? If not, throw an exception in case.

Reply:
It works with colocated synapses, I have added a test.